cmd/go-tpc/tpcc.go

```diff
@@ -6,30 +6,40 @@ import (
 	"net/http"
 	_ "net/http/pprof"
 	"os"
+	"strings"
 
 	"github.com/pingcap/go-tpc/tpcc"
 	"github.com/spf13/cobra"
 )
 
-var tpccConfig tpcc.Config
+var (
+	tpccConfig      tpcc.Config
+	specifiedTables string
+)
 
-func executeTpcc(action string, args []string) {
+func executeTpcc(action string) {
 	if pprofAddr != "" {
 		go func() {
 			http.ListenAndServe(pprofAddr, http.DefaultServeMux)
 		}()
 	}
 
+	chooseTable := len(specifiedTables) > 0
```

Comment:
IMO, chooseTable isn't necessary here.
And it's maybe better to move code 40-42L before 28L, and let `specifiedTables` end it's life early :)